### PR TITLE
Update-eslint-settings

### PR DESCRIPTION
**Pull request by yigitakbulut**
This should fix the unexpected token issue that happened with latest change

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1351,7 +1351,7 @@
     },
     "fs-extra": {
       "version": "2.0.0",
-      "resolved": "https://registry.npmjs.org/fs-extra/-/fs-extra-2.0.0.tgz",
+      "resolved": "https://nexus3.linkurious.net/repository/npm/fs-extra/-/fs-extra-2.0.0.tgz",
```
Comment (davidrapin) [inline package-lock.json:1354]: @yigitakbulut can you revert that? this is a public github repo, it should not reference our private npm repo

File: package-lock.json [hunk 2]
```diff
@@ -2130,7 +2130,7 @@
     },
     "jsonfile": {
       "version": "2.4.0",
-      "resolved": "https://registry.npmjs.org/jsonfile/-/jsonfile-2.4.0.tgz",
+      "resolved": "https://nexus3.linkurious.net/repository/npm/jsonfile/-/jsonfile-2.4.0.tgz",
```
Comment (davidrapin) [inline package-lock.json:2133]: @yigitakbulut can you revert that? this is a public github repo, it should not reference our private npm repo

**Review by davidrapin — CHANGES_REQUESTED** [review]
2 changes requested

**Comment by yigitakbulut** [comment]
> 2 changes requested

@davidrapin ready for post-review 😄

**Pull request merged**